main.go

```diff
@@ -76,7 +70,7 @@ var (
 			Argument:  prefix,
 			Shorthand: "p",
 			Default:   "",
-			Usage:     "the prefix to append to the metric name",
+			Usage:     "the string to append to the metric name",
```

Comment:
Is it not "the string to prepend to the metric name" or "the string to prefix the metric name with"?
Sorry I don't mean to be pedantic...

Reply:
Looked at outdated commit; sorry

Reply:
lmfaoooooo